dali/kernels/imgproc/color_manipulation/hsv_cpu.h

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_KERNELS_IMGPROC_COLOR_MANIPULATION_HSV_CPU_H_
+#define DALI_KERNELS_IMGPROC_COLOR_MANIPULATION_HSV_CPU_H_
+
+#include <utility>
+#include "dali/util/ocv.h"
+#include "dali/core/common.h"
+#include "dali/core/convert.h"
+#include "dali/core/geom/box.h"
+#include "dali/core/error_handling.h"
+#include "dali/kernels/kernel.h"
+#include "dali/pipeline/data/types.h"
+
+namespace dali {
+namespace kernels {
+
+namespace hsv {
+
+constexpr size_t ndims = 3;
```

Comment:
Changed it to `kNdims`, like google style suggests. I'd like to keep it as named variable, so that it's obvious, that it's number of dimensions. If it would be `3`, it wouldn't be that obvious.